examples/cp-all-in-one-cloud/docker-compose.yml

```diff
@@ -0,0 +1,173 @@
+---
+version: '2'
+services:
+
+  schema-registry:
+    image: confluentinc/cp-schema-registry:5.0.0
+    hostname: schema-registry
+    container_name: schema-registry
+    ports:
+      - '8085:8085'
+    environment:
+      SCHEMA_REGISTRY_HOST_NAME: schema-registry
+      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:8085
+      SCHEMA_REGISTRY_KAFKASTORE_SSL_ENDPOINT_IDENTIFIED_ALGORITHM: "https"
+      SCHEMA_REGISTRY_KAFKASTORE_REQUEST_TIMEOUT_MS: 20000
+      SCHEMA_REGISTRY_KAFKASTORE_RETRY_BACKOFF_MS: 500
+      SCHEMA_REGISTRY_KAFKASTORE_SECURITY_PROTOCOL: "SASL_SSL"
+      SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: $SR_BOOTSTRAP_SERVERS
+      SCHEMA_REGISTRY_KAFKASTORE_SASL_JAAS_CONFIG: $SASL_JAAS_CONFIG
+      SCHEMA_REGISTRY_KAFKASTORE_SASL_MECHANISM: "PLAIN"
+
+  ksql-server:
+    image: confluentinc/cp-ksql-server:5.0.0
+    hostname: ksql-server
+    container_name: ksql-server
+    ports:
+      - "8089:8089"
+    environment:
+      KSQL_HOST_NAME: ksql-server
+      KSQL_CONFIG_DIR: "/etc/ksql"
+      KSQL_LOG4J_OPTS: "-Dlog4j.configuration=file:/etc/ksql/log4j-rolling.properties"
+      KSQL_LISTENERS: "http://0.0.0.0:8089"
+      KSQL_AUTO_OFFSET_RESET: "earliest"
+      KSQL_COMMIT_INTERVAL_MS: 0
+      KSQL_CACHE_MAX_BYTES_BUFFERING: 0
+      KSQL_KSQL_SCHEMA_REGISTRY_URL: "http://schema-registry:8085"
+      KSQL_BOOTSTRAP_SERVERS: $BOOTSTRAP_SERVERS
+      KSQL_SECURITY_PROTOCOL: "SASL_SSL"
+      KSQL_SASL_JAAS_CONFIG: $SASL_JAAS_CONFIG
+      KSQL_SASL_MECHANISM: "PLAIN"
+      KSQL_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM: "HTTPS"
+      KSQL_KSQL_STREAMS_PRODUCER_RETRIES: 2147483647
```

Comment:
why do we need to set this here and in the delta?

(same q for other duplicated params)

Reply:
@norwood : only one delta file matters, the `env.delta`.  The others are a by-product of the script, which admittedly is a copy/paste from https://github.com/confluentinc/quickstart-demos/blob/5.0.0-post/ccloud/ccloud-generate-cp-configs.sh

This duplicate code is a pain to keep in sync.  If you have a smarter suggestion for how to pull it in, I'd be happy to change this.

Reply:
I could use submodules, e.g. `git submodule add https://github.com/confluentinc/quickstart-demos` ...

Reply:
The other option is to trim it down so it just writes the `env.delta` file which is used for this demo, since truly none of the other files are used at all with Docker